packages/cli/src/utils/runCommandSync.ts

```diff
@@ -0,0 +1,74 @@
+import chalk from 'chalk'
+import type { ChildProcess, ExecException } from 'child_process'
+import { execSync } from 'child_process'
+
+type ExecSyncError = (ExecException & ChildProcess) | undefined
+
+const showError = ({
+  message,
+  cmd,
+  error,
+  debug,
+}: {
+  message: string
+  cmd: string
+  error: ExecSyncError
+  debug: boolean
+}) => {
+  console.log(`${chalk.red('error')} - ${message}`)
+
+  if (debug && cmd && error) {
+    console.log(`${chalk.magenta('DEBUG')} - $ ${cmd} error root ↓`)
+    console.log(error.stdout?.toString())
+  }
+
+  process.exit(1)
+}
+
+const showWarning = ({
+  message,
+  cmd,
+  error,
+  debug,
+}: {
+  message: string
+  cmd: string
+  error: ExecSyncError
+  debug: boolean
+}) => {
+  console.log(`${chalk.yellow('warn')} - ${message}`)
+
+  if (debug && cmd && error) {
+    console.log(`${chalk.magenta('DEBUG')} - $ ${cmd} warn root ↓`)
+    console.log(error.stdout?.toString())
+  }
+}
+
+export const runCommandSync = ({
+  cmd,
+  errorMessage,
+  throws,
+  debug,
+  cwd
+}: {
+  cmd: string
+  errorMessage: string
+  throws: 'warning' | 'error'
+  debug: boolean
+  cwd?: string
+}) => {
+  try {
+    execSync(debug ? `${cmd} --debug --verbose 2>&1` : `${cmd} 2>&1`, {
```

Comment:
out of curious: what is this `2>&1`?

Reply:
The error output directed to the same "location" as the regular output, which is helpful when we need to log errors in debug mode.